roles/system_configuration.role

```diff
@@ -0,0 +1,6 @@
+nuclear_role(
+  # FileWatcher, ConsoleLogHandler and Signal Catcher Must Go First.
```

Comment:
should we update this comment in roles that deliberately don't include those modules?